book-list.html

```diff
@@ -0,0 +1,140 @@
+<!doctype html>
+
+<html lang="en"> 
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="master-container">
+    <div class="navbar-container">
+      <div class="border">
```

Comment:
Is this class necessary? You can apply the border directly to `navbar-container`